Tutorial.md

```diff
@@ -38,9 +38,9 @@ As this tutorial is intended to illustrate the management of data for phylogenet
 
 #### 2.1.1 Creating and Curating Data in CLDF
 
-While the details and advantages of CLDF have been discussed and presented in Forkel et al. (2018), we would like to briefly highlight the benefits of using CLDF as a basis for datasets in historical linguistics and, by extension, phylogenetic analyses. While, at a first glance, CLDF and the outer appearance of a dataset (e.g. information split across multiple files), might seem daunting or even plainly unnecessary, it gives researchers a powerful toolset via the expressiveness of relational data structures in well-supported serialization formats, namely CSV and JSON. The format allows for the creation of truly linked data (see for example the embedded information concerning languoids and concepts in the Kho-Bwa sample dataset), which at the same time exists independently of any other software stack, given that all the information are stored in plain (but structured) text files. The fact that multiple files are 'glued' together by a JSON file, which specifies how files are related as well as metadata information for every file and column, allows for unique identifiers for every record, and guarantees maximum flexibility with regards to research designs and experimental setups.
+While the details and advantages of CLDF have been discussed and presented in Forkel et al. (2018), we would like to highlight the benefits of using CLDF as a basis for datasets in historical linguistics and, by extension, phylogenetic analyses. While, at a first glance, CLDF and the outer appearance of a dataset (e.g. information split across multiple files), might seem daunting or even plainly unnecessary, it gives researchers a powerful toolset via the expressiveness of relational data structures in well-supported serialization formats, namely CSV and JSON. The format allows for the creation of truly linked data (see for example the embedded information concerning languoids and concepts in the Kho-Bwa sample dataset), which at the same time exists independently of any other software stack, given that all the information are stored in plain (but structured) text files. The fact that multiple files are 'glued' together by a JSON file, which specifies how files are related as well as metadata information for every file and column, allows for unique identifiers for every record, and guarantees maximum flexibility regarding research designs and experimental setups.
```

Comment:
While the details and advantages of CLDF have been discussed and presented in Forkel et al.
(2018), we would like to highlight the benefits for historical linguistics and, by extension,
phylogenetic analyses. 
While at first glance CLDF might seem daunting or unnecessary, it has many benefits. First,
the structure is easily importable into standard statistical and analytic tools in many
programming languages or analysis packages. Second, CLDF gives researchers a powerful toolset
to express relational data structures in well-supported open formats, namely CSV and JSON. 
Third, the metadata contained within CLDF describes the data structure for enabling permanent
documentation of the data structure, and facilitating the re-use of the data. Further, the
metadata allows known types of columns to be validated and checked for incorrect content
ensuring that the data is correct. Fourth, CLDF is linked data which means that any type of
extra information can be easily added to the dataset by linking it to the relevant identifier 
(see for example the embedded information concerning languoids and concepts in the Kho-Bwa
sample dataset). These data exist independently of any other software stack as all they can
be stored in plain (but structured) text files. The fact
that multiple files are 'glued' together by a JSON file, which specifies how files are
related as well as metadata information for every file and column, allows for unique
identifiers for every record, and guarantees maximum flexibility regarding research designs
and experimental setups.